blackberggroup/VA-DOE
# Feature/jls 6158463071/dta client content changes

## What type of PR is this? (check all applicable)


## Description
Added background gradients to break up white space
Add "Diffusion Talent Accelerator Phases" title to 
Copy edit, 1st bullet: January - March
Delete “Empower Your Impact”
In Specialists, rm duplicate location and add "Capstone Project:" to the project line

## Related Tickets & Documents
- Monday Task #6158463071
https://blackberg-group.monday.com/boards/1931719392/pulses/6158463071

## QA Instructions, Screenshots, Recordings
Make sure gradients aren't visually distracting especially when resizing (removed at mobile and tablet breakpoints)
Confirm copy changes

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_


## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized 

## Changed Files in this Pull Request ##
- _includes/talent-accelerator/dta-syllabus.html
 - _includes/talent-accelerator/impact.html
 - _includes/talent-accelerator/mission.html
 - _includes/talent-accelerator/timeline.html
 - _site/assets/css/styles.css
 - _site/assets/js/mustache-fellows-render.js
 - _site/assets/sass/vha/pages/talent-accelerator/mission/_index.scss
 - _site/assets/sass/vha/pages/talent-accelerator/timeline/_index.scss
 - _site/diffusion-talent-accelerator/index.html
 - assets/css/styles.css
 - assets/js/mustache-fellows-render.js
 - assets/sass/vha/pages/talent-accelerator/mission/_index.scss
 - assets/sass/vha/pages/talent-accelerator/timeline/_index.scss
